js/styles.js

```diff
@@ -109,3 +109,15 @@ function unhighlightCounty() {
     .classed("highlighted-county", false);
 }
 
+function scaleCircleStroke(selection, scale) {
```

Comment:
I don't particularly like `map-utils` because it's too big and is just a catch all for things since almost everything is changing the map. I'd like to avoid that in the future and be more specific about the files.